From 7daca56050529176b96126c7c3cb84109705331d Mon Sep 17 00:00:00 2001
From: Vol Zhdanov <wight554@gmail.com>
Date: Wed, 26 Sep 2018 16:48:46 +0000
Subject: [PATCH 303/420] dts: xiaomi: msm8998: update chiron batterydata

* extracted from MIMIX2Global_V10.0.2.0 DTS
* fixes bug with battery drops on firmware > 8.4.19

Change-Id: If2526d8a43dd19a90efe88bf94f4e0c70b40de6b
---
 .../xiaomi/batterydata-chiron-atl-3400mAh.dtsi     | 72 +++++++++++-----------
 .../xiaomi/batterydata-chiron-cos-3400mAh.dtsi     | 72 +++++++++++-----------
 2 files changed, 72 insertions(+), 72 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-atl-3400mAh.dtsi b/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-atl-3400mAh.dtsi
index f505e3174757..8748e0faee3c 100644
--- a/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-atl-3400mAh.dtsi
+++ b/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-atl-3400mAh.dtsi
@@ -8,46 +8,46 @@ qcom,chiron_atl_3400mah {
 	qcom,batt-id-kohm = <68>;
 	qcom,battery-beta = <3380>;
 	qcom,battery-type = "chiron_atl";
-	qcom,checksum = <0xE38B>;
+	qcom,checksum = <0x8E3D>;
 	qcom,gui-version = "PMI8998GUI - 2.0.0.58";
 	qcom,fg-profile-data = [
-			6C 1F B7 05
-			69 0A AD FC
-			7B 1D 4A 07
-			48 12 15 14
-			7E 18 52 23
-			F9 44 C2 53
-			55 00 00 00
-			0E 00 00 00
-			00 00 FC 07
-			C6 C5 D8 C2
-			20 00 08 00
-			04 D3 CA E4
-			F7 F4 CB EA
-			77 01 C7 0B
-			05 FC D9 2A
-			18 06 09 20
+			0A 1F 3A 06
+			00 0A C7 06
+			D4 1C 0C 02
+			98 0D C7 0A
+			95 17 D6 23
+			7F 44 31 5A
+			6F 00 00 00
+			11 00 00 00
+			00 00 D8 BB
+			61 CD DF A2
+			26 00 08 00
+			AB EA 51 EC
+			D1 FC BB F3
+			58 0C F3 FA
+			62 06 BB 32
+			1F 06 09 20
 			27 00 14 00
-			48 1D 4E E3
-			8D FA 54 CC
-			EF 1C 68 02
-			3F 0D 89 0A
-			FA 19 D3 1B
-			3E 3D B2 4A
-			70 00 00 00
-			12 00 00 00
-			00 00 05 CC
-			D8 C2 27 C3
-			1B 00 00 00
-			D7 EA CA E4
-			26 EC 52 E3
-			85 00 70 03
-			83 03 0F 1B
-			B3 33 CC FF
+			18 20 25 05
+			D4 0A F4 05
+			C8 1C ED 02
+			1A 0C 19 12
+			91 18 AC 23
+			20 44 D6 5A
+			6E 00 00 00
+			0E 00 00 00
+			00 00 86 D5
+			3E CA 24 CA
+			1E 00 00 00
+			5B F2 51 EC
+			7B F4 22 EB
+			7A 07 87 FB
+			69 BA BC 1B
+			AB 33 CC FF
 			07 10 00 00
-			4A 0D 66 46
-			1B 00 40 00
-			49 01 0A FA
+			3E 0D 66 46
+			1E 00 40 00
+			66 01 0A FA
 			FF 00 00 00
 			00 00 00 00
 			00 00 00 00
diff --git a/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-cos-3400mAh.dtsi b/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-cos-3400mAh.dtsi
index ddc54107a30e..36c2af0817bd 100644
--- a/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-cos-3400mAh.dtsi
+++ b/arch/arm/boot/dts/qcom/xiaomi/batterydata-chiron-cos-3400mAh.dtsi
@@ -8,46 +8,46 @@ qcom,chiron_cos_3400mah {
 	qcom,batt-id-kohm = <100>;
 	qcom,battery-beta = <3380>;
 	qcom,battery-type = "chiron_cos";
-	qcom,checksum = <0x5B7D>;
+	qcom,checksum = <0x1313>;
 	qcom,gui-version = "PMI8998GUI - 2.0.0.58";
 	qcom,fg-profile-data = [
-			80 1F 9B 05
-			7F 0A 89 FC
-			69 1D 47 EC
-			F8 0B 8B 14
-			95 19 17 22
-			FE 3C E7 4A
-			57 00 00 00
-			17 00 00 00
-			00 00 E8 C4
-			AC C5 06 BB
-			1F 00 08 00
-			1C DA 06 E5
-			BD F5 22 E3
-			A7 F2 E9 03
-			0C 04 38 1D
-			19 06 09 20
-			27 00 14 00
-			14 1E 37 F5
-			95 02 7F F5
-			EF 1C 5A 02
-			4B 0D 98 0A
-			1C 1A 71 1B
-			B7 3D 06 4A
-			70 00 00 00
+			A1 20 86 04
+			28 0B C6 05
+			01 1D 7B 01
+			47 05 43 03
+			A3 17 C6 23
+			A1 44 0E 5A
+			8C 00 00 00
 			11 00 00 00
-			00 00 A9 CC
-			89 BB 9D BA
-			1A 00 00 00
-			73 EA 06 E5
-			BA EC 2C 00
-			5D FA 47 0A
-			81 EA A4 1A
-			A1 33 CC FF
+			00 00 FB BC
+			CA CD D1 BA
+			26 00 08 00
+			A2 DB AA E4
+			43 FC 28 01
+			61 0C 6C 01
+			5C 06 41 32
+			21 06 09 20
+			27 00 14 00
+			96 20 97 04
+			34 0B A3 05
+			D8 1C AD 02
+			9A 0C 98 0B
+			90 18 B6 23
+			11 44 E1 5A
+			76 00 00 00
+			0E 00 00 00
+			00 00 AE D5
+			2A CA 82 C3
+			1E 00 00 00
+			6E EA AA E4
+			8F FD B2 00
+			44 ED 03 03
+			BD 03 48 22
+			C3 33 CC FF
 			07 10 00 00
-			34 0D 66 46
-			1A 00 40 00
-			49 01 0A FA
+			33 0D 66 46
+			1E 00 40 00
+			60 01 0A FA
 			FF 00 00 00
 			00 00 00 00
 			00 00 00 00
-- 
2.15.0

